WooCommerce/Classes/ViewRelated/Dashboard/Settings/SettingsViewController.swift

```diff
@@ -379,6 +391,15 @@ private extension SettingsViewController {
         show(viewController, sender: self)
     }
 
+    func cardReadersV2WasPressed() {
+        ServiceLocator.analytics.track(.settingsCardReadersTapped)
+        guard let viewController = UIStoryboard.dashboard.instantiateViewController(ofClass: CardReaderSettingsPresentingViewController.self) else {
+            fatalError("Cannot instantiate `CardReaderSettingsPresentingViewController` from Dashboard storyboard")
+        }
+        viewController.configure(viewModelsAndViews: [])
```

Comment:
In a follow on issue, I think we'll want to define a DependencyContainer registers each of the viewmodels (and related view class names) and can serve as a source for the viewModelsAndViews here.

No need for SettingsViewController to know all those details.

Reply:
Makes sense. Also, it might make sense to encapsulate the details abut how the view controller is instantiated in a static method in the view controller itself.